include/highfive/h5easy_bits/H5Easy_scalar.hpp

```diff
@@ -94,32 +92,17 @@ struct dump_extend_impl
         file.flush();
         return dataset;
     }
-};
-
-// load "scalar" as part of a larger DataSet
-template <class T>
-struct load_impl
-{
-    static T run(const File& file,
-                 const std::string& path,
-                 const std::vector<size_t>& idx)
-    {
+    // load a part of a larger DataSet
+    static T
+    load_part(const File& file, const std::string& path, const std::vector<size_t>& idx) {
         std::vector<size_t> ones(idx.size(), 1);
         DataSet dataset = file.getDataSet(path);
         T data;
         dataset.select(idx, ones).read(data);
         return data;
     }
-};
-
-}  // namespace scalar
-
-// load from DataSet
-template <class T, class E = void>
-struct load_impl
-{
-    static T run(const File& file, const std::string& path)
-    {
+    // load entire DataSet
```

Comment:
I think it always a good idea to keep whitelines between functions

Reply:
Understood. I just used `.clang-format`.